payroll/models/hr_contract.py

```diff
@@ -38,8 +38,5 @@ def get_all_structures(self):
                  hierachy (parent=False first, then first level children and
                  so on) and without duplicates
         """
-        structures = self.mapped("struct_id")
-        if not structures:
-            return []
-        # YTI TODO return browse records
-        return list(set(structures._get_parent_structure().ids))
+        # TODO: remove, too simple and not used
```

Comment:
This TODO is asking to remove the line you just changed or  im understanding it wrong?

Reply:
This method can be removed, as it implements so useful logic at all.
But I hesitated removing it, for backward compatibility.